osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -647,7 +648,7 @@ public void DeleteAll()
 
             if (maps.Count == 0) return;
 
-            var notification = new ProgressNotification
+            var notification = new ProgressNotification("")

```

Comment:
How about making the constructor parameter optional? Would fit well with the constructor of `Notification` too.